### ISSUE TITLE
LPD biasses do not follow the original paper

### ISSUE THREAD

**Issue by D1rk123** [CambridgeCIA/LION#139]
In the LPD implementation the bias terms of the convolution layers are turned off (bias=False in nn.Conv2d). However, in the original LPD paper they say that the convolution layers do have biasses. (Section IV.A In the equation after the line "where the k:th component is given by"). I also checked the original code, and it does not specify the bias, but the Tensorflow default is True.

Was it a deliberate choice to not follow the paper? Otherwise I think it's better to change the code so that bias=True for all convolution layers.

**Comment by D1rk123** [comment]
Thanks for your quick reply!

Having a separate original paper settings, and LION default settings sounds like a good idea to me. In my (somewhat limited) experiments, LPD also performed better without convolution biasses. In the pull request I didn't include the two versions of the settings yet.

**Comment by AnderBiguri** [comment]
Yup, will likely do that. I hope to tidy all this soon and get some
examples and demos by the end of the year, so it gets easier to use. Let me
know if you have any questions meanwhile, as its all a bit in the middle of
construction now
